### ISSUE TITLE
Error with angular@4.0.0-beta.7 

### ISSUE THREAD

**Issue by kavi87** [formly-js/ng-formly#382]
Hi ! 

**I'm submitting a ...** 
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request
```
**Current behavior**

The example on the README does not work with angular 4 beta ("Cannot read property "templateOptions" of undefined). Everything is fine with 2.4.8.

**Expected behavior**

To work with angular 4

**Comment by aitboudad** [comment]
seems already fixed in master branch with #360 and it should be landed in the next release.

**Comment by aitboudad** [comment]
@mohammedzamakhan can you publish a new version or just add me to `npm` and I'll take care of it.
Thanks

**Comment by ampsarfraz** [comment]
hi .. any ETA on this? Please let us know. Thanks

**Comment by aitboudad** [comment]
in the meantime you can use `npm install --save formly-js/ng-formly`

**Comment by mohammedzamakhan** [comment]
@aitboudad I added you on npm 🎉🎉

**Comment by aitboudad** [comment]
thanks!

**Comment by aitboudad** [comment]
@kavi87 @ampsarfraz a new version is released (`1.0.0-rc.2`) that support angular 4.
